src/Explorer/Controls/Settings/SettingsUtils.tsx

```diff
@@ -0,0 +1,42 @@
+export interface StatefulValue<T> {
+  baseline: T;
+  current: T;
+  isValid: boolean;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Will move most of it to unknown. There are couple of place where I still have to use any, since it deal with indexingPolicy returned from the container which has 'any' type.

Reply:
In that case we should fix the indexingPolicy type to be correct.